### DISCUSSION TITLE
Supporting Kompendium

### DISCUSSION THREAD

**Discussion by kvanry** [bkbnio/kompendium#169] [General]
I'd like to get involved more long term, but short term, really want to show my appreciation for the work done on Kompendium and keep Kompendium as a long term solution for Ktor Swagger UI. Are there ways to donate or other ways to keep the spirits of this project alive and growing?

**Comment by brizzbuzz** [comment]
That means a lot! I started Kompendium because I needed a simple way to generate documentation in an automated way, and I'm really glad other people find use out of it. 

But no donations are necessary! I had a lot of fun building it, and that's enough for me. Hearing that it's helping other people makes me super happy :) 

As for the long term, there are a couple ways to help in if you're interested 

1. V2 was _just_ released, so any gaps in documentation or bugs you find, helping to patch those would be awesome 
2. Ktor 2 is in beta and is a major refresh, so the plan is to have a new repo Kompendium 2 that has the same approach as this repo, but geared towards Ktor 2.  If you would like, I could invite you to that repo. Haven't really started much work on it yet, but should be fun to take the best of this repo and merge with the best of what Jetbrains is working on.  

With that said, I _do_ hope that ultimately the team over at Jetbrains decides to offer this capability natively in Ktor itself. But until that day let's make Kompendium as great as we can!
